shared-module/usb/core/Device.c

```diff
@@ -34,15 +34,30 @@
 #include "shared-bindings/usb/core/__init__.h"
 #include "shared-module/usb/utf16le.h"
 #include "supervisor/shared/tick.h"
+#include "supervisor/usb.h"
 
+// Track what device numbers are mounted. We can't use tuh_ready() because it is
+// true before enumeration completes and TinyUSB drivers are started.
+STATIC size_t _mounted_devices = 0;
+
+void tuh_mount_cb(uint8_t dev_addr) {
+    _mounted_devices |= 1 << dev_addr;
```

Comment:
are device addresses guaranteed to be small numbers <32?

Reply:
I believe so because we configure TinyUSB for 4 devices and a hub. https://github.com/hathach/tinyusb/blob/f1e006d09bd32088ab421d0b519eb89c531eda4e/src/host/usbh.c#L199

Reply:
https://github.com/hathach/tinyusb/blob/master/src/host/usbh.c#L1504